mitodl/ocw-studio
# File Upload UI

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #92 

#### What's this PR do?
Adds a basic form input to the add/edit resource form for uploading a file.

#### How should this be manually tested?
- Add a new resource and include a file.  It should save without errors.
- Edit that resource, you should see the uploaded filename as a working hyperlink.  Choose another file and submit, it should save without errors.
- The add/edit page forms should look/work the same, with no file field.

#### Screenshots (if appropriate)
<img width="1138" alt="Screen Shot 2021-03-11 at 11 45 38 AM" src="https://user-images.githubusercontent.com/187676/110822974-9fd56b00-825f-11eb-8b34-82ef593e763c.png">

<img width="1143" alt="Screen Shot 2021-03-11 at 11 47 22 AM" src="https://user-images.githubusercontent.com/187676/110822984-a368f200-825f-11eb-804f-ab2113ea6482.png">


#### What GIF best describes this PR or how it makes you feel?
(Optional)
